### ISSUE TITLE
Amazing Tool

### ISSUE THREAD

**Issue by akaAMAZING** [xanthics/PoE_Weighted_Search#28]
Hi there xanthics, first of all - amazing tool!

Have been using it for a while now and each time that I do, I wonder if it would be possible to do the same thing for rings, amulets and other pieces of gear. I usually just use the jewel search and edit the "Item Category" but not all of the possible mods are tested.

Is there something I could do to be able to use it for those purposes? Like a separate tool for each item slot. Is this something you had on your scope? If not, could you kindly explain (eli5) how I would go about achieving a result?

Thank you!

**Comment by xanthics** [comment]
![image](https://user-images.githubusercontent.com/3137623/106305132-2898ca00-6211-11eb-8567-0b507e2299d4.png)
 It's already implemented.  Mods enabled can be found [here](https://github.com/xanthics/PoE_Weighted_Search/blob/master/restrict_mods.py)
